composer/algorithms/__init__.py

```diff
@@ -16,6 +16,8 @@
 from composer.algorithms.cutmix import CutMixHparams as CutMixHparams
 from composer.algorithms.cutout import CutOut as CutOut
 from composer.algorithms.cutout import CutOutHparams as CutOutHparams
+from composer.algorithms.dropblock import DropBlock as DropBlock
```

Comment:
Should ShakeDrop and ShakeShake be in here?

Reply:
Rationale is that we are pulling in Algorithms, whereas the SD and SS objects are nn.Modules

Reply:
Ah makes sense, thanks!